CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the create-aptos-dapp tool will be captured in this file.
 # Unreleased
 
 - Add support to define a client API Key
+- Remove `uniqueHolders` from token minting dapp template due to inefficient indexer query
```

Comment:
this has already been fixed https://github.com/aptos-labs/create-aptos-dapp/pull/259